pkg/web_app/lib/src/google_auth_js.dart

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+@JS('gapi.auth2')
```

Comment:
Please leave a documentation comment at the library level, saying this is partial bindings for:
https://developers.google.com/identity/sign-in/web/reference